src/Raven.Client/Documents/Operations/Revisions/DeleteRevisionsOperation.cs

```diff
@@ -33,8 +33,8 @@ public sealed class DeleteRevisionsOperation : IMaintenanceOperation<DeleteRevis
         /// </exception>
 
         /// <param name="removeForceCreatedRevisions">
-        /// Indicates whether to include force-created revisions in the deletion 
-        /// (Can find more info <see href="https://ravendb.net/docs/article-page/6.0/csharp/document-extensions/revisions/overview#force-revision-creation">here</see>).
+        /// Indicates whether to include force-created revisions in the deletion. 
+        /// (Can find more info <see href="https://ravendb.net/docs/article-page/latest/csharp/document-extensions/revisions/overview#force-revision-creation">here</see>).
```

Comment:
This should link to https://github.com/ravendb/ravendb/blob/v6.2/src/Raven.Client/DocumentationUrls.cs file and be set to exact version, 6.2 in this case

Reply:
done, fixed in all places